test/test.js

```diff
@@ -473,6 +491,23 @@ describe('YeldaChat', () => {
   })
 
   describe('yeldaChat.resetChat', () => {
+
+    it('should load data of the second call', async () => {
+      yeldaChat.resetChat(Object.assign({}, validMockData, { locale: 'fr_FR' }))
+      await yeldaChat.resetChat(Object.assign({}, validMockData, { locale: 'en_US' }))
+
+      expect(yeldaChat.locale).to.equal('en_US')
```

Comment:
```js
    it('should use data friom the latest call to build iframe url', async () => {
      yeldaChat.resetChat(Object.assign({}, validMockData, { locale: 'fr_FR' }))
      await yeldaChat.resetChat(Object.assign({}, validMockData, { locale: 'en_US' }))

      const result = 'https://app.yelda.ai/chat?assistantId=12345678&assistantSlug=testClient&locale=en_US&location=https%3A%2F%2Fyelda.ai%2F'
     expect(yeldaChat.webChatIframe.getAttribute('src')).to.deep.equal(result)

```

this kind of check would really test if the webchat has been really loaded with the latest call settings.
Checking global variable is not a real "proof"